.licenserc.yml

```diff
@@ -0,0 +1,20 @@
+header:
+  license:
+    spdx-id: BSD-3-Clause
+    copyright-owner: UChicago Argonne, LLC
+    content: |
+        Copyright 2019 UChicago Argonne, LLC.
+        (c.f. AUTHORS, LICENSE)
+
+        This file is part of the NRM project.
+        For more info, see https://xgitlab.cels.anl.gov/argo/nrm
```

Comment:
let's go ahead and fix that url to point to the new repo

Reply:
Sounds good, but which one? nrm-python, nrm-core, libnrm, etc.

Reply:
nrm-python for this repo is good